flash/website/templates/home.html

```diff
@@ -108,7 +102,31 @@ <h4 style="margin: 0;">All cards</h4>
 <script>
   $(document).ready(function() {
     $('.carousel').carousel({
-      interval: 5000 // Pięć sekund
+      interval: 5000
+    });
+
```

Comment:
O jaki ładny ajaksik :D

Reply:
Mi się podoba, bo wolę po staremu, ale teraz w modzie promisy - https://stackoverflow.com/questions/39751395/whats-the-difference-between-promise-and-ajax